vendor/assets/javascripts/koi/clipboard.js

```diff
@@ -1,7 +1,944 @@
 /*!
- * clipboard.js v1.5.16
- * https://zenorocha.github.io/clipboard.js
+ * clipboard.js v2.0.6
+ * https://clipboardjs.com/
```

Comment:
not sure if it's worth having javascript that's only used by the styleguide.
perhaps the styleguide could be removed from koi completely?

Reply:
also looks like the latest version uses `new ClipboardJS()` rather than `new Clipboard()` which the styleguide uses

Reply:
Removed styleguide and clipboard.js